brownie/network/rpc.py

```diff
@@ -372,3 +387,38 @@ def _check_connections(proc: psutil.Process, laddr: Tuple) -> bool:
         return laddr in [i.laddr for i in proc.connections()]
     except psutil.AccessDenied:
         return False
+
+
+def _validate_cmd_settings(cmd_settings: dict) -> dict:
+    CMD_TYPES = {
+        "port": int,
+        "gas_limit": int,
+        "block_time": int,
+        "time": datetime.datetime,
+        "accounts": int,
+        "evm_version": str,
+        "mnemonic": str,
+        "account_keys_path": str,
+        "fork": str,
+    }
+    for cmd, value in cmd_settings.items():
+        if (
+            cmd in CLI_FLAGS.keys()
+            and cmd in CMD_TYPES.keys()
+            and not type(value) == CMD_TYPES[cmd]
+        ):
+            raise ValueError(
+                f'Wrong type for cmd_settings "{cmd}" ({value}). '
+                f"Found {type(value)}, but expected {CMD_TYPES[cmd]}."
+            )
+
+    if "default_balance" in cmd_settings:
+        try:
+            cmd_settings["default_balance"] = int(cmd_settings["default_balance"])
```

Comment:
Make sure this is well documented, since everywhere else in the config we assume an integer value to be denominated in wei.

Reply:
The reasoning behind this is that ganache wants the argument in ether. I wanted to keep the parameters as close to the ganache-cli parameters as possible. It's your call if you want to keep a strict "for brownie, input in ether is in wei units" approach. Let me know

Reply:
I agree with your approach, I just wanted to stress the importance of documentation on this :)